### DISCUSSION TITLE
Blazor Isolated CSS in Razor Component Library Ignored

### DISCUSSION THREAD

**Discussion by marqdouj** [dotnet/maui#1968] [Q&A]
When I add an isolated *.css file to a razor component in a class library the css is ignored. If I copy the css to the app.css file it works fine.
Is this by design?

**Comment by nlipkanov** [comment]
In my case, whenever I create an isolated *.razor.css for a razor component, I am getting an error and the MAUI application cannot start.
Do you experienced something similar to this?
